lib/Parser/Scan.h

```diff
@@ -667,7 +667,7 @@ class Scanner : public IScanner, public EncodingPolicy
     BOOL m_doubleQuoteOnLastTkStrCon :1;
     bool m_OctOrLeadingZeroOnLastTKNumber :1;
     BOOL m_fSyntaxColor : 1;            // whether we're just syntax coloring
```

Comment:
Yeah... that get rids of the warning.
I guess the rest of the fields should be change to bool too..... but those didn't cause warnings, so I didn't touch it.